frontend/src/App.tsx

```diff
@@ -1,9 +1,38 @@
 import "./App.css";
+import { useState } from "react";
+
+import Input from "./components/Input";
+import TodoList from "./components/TodoList";
+import { Todo } from "./models/todo.model";
 
 function App() {
+  const [todos, setTodos] = useState<Todo[]>([
+    { description: "Hello world", id: "244454", isDone: false },
+    { description: "Hello Pakistan", id: "24454", isDone: false },
+    { description: "Hello GB", id: "24433", isDone: false },
+  ]);
+
+  let value: string = "";
```

Comment:
`value` is a vague name. `newTodo` is more appropriate and tells that this holds the description for a new Todo. Also, this should be a state variable, not a regular `let`/`const` variable, as we want the component to re-render as this changes.